sdk/servicebus/service-bus/migrationguide.md

```diff
@@ -1,33 +1,84 @@
 # Guide to migrate from @azure/service-bus v1 to v7
 
-This document is intended for users that would like to try out preview 8
-for @azure/service-bus. As the package is in preview, these details might
-change as the package is developed before its final release.
+This guide is intended to assist in the migration from version 1 of the Service Bus client library `@azure/service-bus` to version 7 of the same library. It will focus on side-by-side comparisons for similar operations between the two packages.
 
-## General changes
+Familiarity with the version 1 of the `@azure/service-bus` library is assumed. For those new to the Service Bus client library for JavaScript, please refer to the [README](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/servicebus/service-bus/README.md) and [Service Bus samples](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/servicebus/service-bus/samples) for the `@azure/service-bus` library rather than this guide.
 
-Version 7 of `@azure/service-bus` provides some "under-the-hood" changes
-by standardizing on common infrastructure with `@azure/event-hubs`. This change
-brings this package in line with the [Azure SDK Design Guidelines for Typescript](https://azure.github.io/azure-sdk/typescript_introduction.html#design-principles).
+## Migration benefits
 
-## API changes from V1 to V7
+A natural question to ask when considering whether or not to adopt a new version or library is what the benefits of doing so would be. As Azure has matured and been embraced by a more diverse group of developers, we have been focused on learning the patterns and practices to best support developer productivity and to understand the gaps that the JavaScript client libraries have.
 
-### Creating ServiceBusClient
+There were several areas of consistent feedback expressed across the Azure client library ecosystem. One of the most important is that the client libraries for different Azure services have not had a consistent approach to organization, naming, and API structure. Additionally, many developers have felt that the learning curve was difficult, and the APIs did not offer a good, approachable, and consistent onboarding story for those learning Azure or exploring a specific Azure service.
 
-- `ServiceBusClient` can now be constructed using new(). The static methods to
-  construct it have been removed.
+To try and improve the development experience across Azure services, including Service Bus, a set of uniform [design guidelines](https://azure.github.io/azure-sdk/general_introduction.html) was created for all languages to drive a consistent experience with established API patterns for all services. A set of [.TypeScript & JavaScript specific guidelines](https://azure.github.io/azure-sdk/typescript_introduction.html) was also introduced to ensure that .NET clients have a natural and idiomatic feel. Further details are available in the guidelines for those interested.
```

Comment:
```suggestion
To try and improve the development experience across Azure services, including Service Bus, a set of uniform [design guidelines](https://azure.github.io/azure-sdk/general_introduction.html) was created for all languages to drive a consistent experience with established API patterns for all services. A set of [TypeScript & JavaScript specific guidelines](https://azure.github.io/azure-sdk/typescript_introduction.html) was also introduced to ensure that .NET clients have a natural and idiomatic feel. Further details are available in the guidelines for those interested.
```

Reply:
And also the ".NET Clients"?
(Same as I mentioned for event-hubs guide)

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-js/pull/12350/commits/07bf6d252e8bca50196fdfb14a0e5a8d3487fa3d